src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/JsonTest/JsonBulkCopyTest.cs

```diff
@@ -232,43 +237,46 @@ private async Task BulkCopyDataAsync(CommandBehavior cb, bool enableStraming)
                     long countEnd = System.Convert.ToInt32(await commandRowCount.ExecuteScalarAsync());
                     _output.WriteLine("Ending row count = {0}", countEnd);
                     _output.WriteLine("{0} rows were added.", countEnd - countStart);
+                    Assert.Equal(expectedTransferCount, countEnd - countStart);
                 }
             }
         }
 
+        [Theory]
         [InlineData(CommandBehavior.Default, false)]
         [InlineData(CommandBehavior.Default, true)]
         [InlineData(CommandBehavior.SequentialAccess, false)]
         [InlineData(CommandBehavior.SequentialAccess, true)]
-        [ConditionalTheory(typeof(DataTestUtility), nameof(DataTestUtility.IsJsonSupported))]
```

Comment:
Can these tests be run on an older SQL Server without JSON support?

Reply:
No. Had issue with using InlineData with ConditionalTheory. Have changed it to ClassData now.